src/components/Post/CommentDialogBox.jsx

```diff
@@ -44,15 +45,18 @@ const CommentDialogBox = ({
             <div key={userComment.id}>
               <div className="commentCard">
                 <div>
-                  <Link
-                    className="comment-doer"
-                    to={`/dummygram/${userComment.content.username}`}
-                  >
-                    <AccountCircleIcon />
-                    <span className="comment-doer-name">
-                      {userComment.content.username}
-                    </span>
-                  </Link>
+                <div className="comment-doer">
+                <AccountCircleIcon />
+                <span className="comment-doer-name" onClick={() => {
+                  navigate("/dummygram/profile", {
+                    state: {
+                      name: userComment.content.username,
```

Comment:
photo and other details can be passed in this state object

Reply:
> photo and other details can be passed in this state object

I tried doing that, but it didn't work, thats why i removed that code in the updated commit :/

![image](https://github.com/narayan954/dummygram/assets/97438986/af25f50d-6b3e-40f2-bb59-4121b608b4a9)

Reply:
ohhhh, perhaps some other issue, or component getting refreshed on its own. But dont worry now, route has been made dynamic and you dont need to pass these states at all now!! Amazing, isn't it? :)